bindings/nodejs/lib/types/wallet/transaction-options.ts

```diff
@@ -46,9 +45,9 @@ export type CustomAddress = {
 export interface CreateNativeTokenParams {
     aliasId?: string;
     /** Hex encoded number */
-    circulatingSupply: HexEncodedAmount;
+    circulatingSupply: bigint;
```

Comment:
You sometimes have `bigint | string` and sometimes just `bigint` for params, what's the reasoning?

Reply:
If the serialization is done manually, for example in `bindings/nodejs/lib/client/client.ts`  the resulted property is converted from `bigint` to `string` type. If we want to make it take `bigint` only, we could make the entire object of `any` type, but then we don't have type safety internally.

Reply:
yea we should make it at least bigint | hexencodedamount   (if that works)